### PR TITLE
Create test and POM for adding a home file system

### PR DESCRIPTION
**Pull request by manfredi**
- Description: Create test and POM for adding a home file system
- Related ticket: https://progress.opensuse.org/issues/133103
- Verification run:
  + [Test result overview - Staging](https://openqa.opensuse.org/tests/overview?build=manfredi%2Fos-autoinst-distri-opensuse%23issues-133103&distri=alp&version=agama-3.0-staging)
  + [Test result overview - Devel](https://openqa.opensuse.org/tests/overview?build=manfredi%2Fos-autoinst-distri-opensuse%23issues-133103&distri=alp&version=agama-3.0-devel)

Related PR: https://github.com/jknphy/e2e-agama-playwright/pull/13

### CODE REVIEW

File: tests/yam/agama/add_home_file_system.pm [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Copyright 2023 SUSE LLC
```
Comment (jknphy) [inline tests/yam/agama/add_home_file_system.pm:1]: could you check if we could use one of the unified files from #17629 
In theory we should just need a single file to trigger this.

Comment (manfredi) [reply tests/yam/agama/add_home_file_system.pm:1]: Do you mean to use the file `tests/yam/agama/agama.pm` by set the _var_ `AGAMA_TEST` to trigger this?

Comment (jknphy) [reply tests/yam/agama/add_home_file_system.pm:1]: yes, that PR still needs more improvements, but if you could try to use it and even give feedback there would help us. But I'm aware it is a bit out of scope, we could also merge this and have more unification tasks (but I prefer less tickets and more PRs :) ) up to you.

File: schedule/yam/agama/add_home_file_system.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: add_home_file_system
+description: >
+  Playwright use add-home-file-system test on agama
```
Comment (jknphy) [inline schedule/yam/agama/add_home_file_system.yaml:4]: Please add a description independent of a name of a file that could potentially change.
s/agama/Agama/ (only for text)

Comment (manfredi) [reply schedule/yam/agama/add_home_file_system.yaml]: Updated

**Pull request merged**